lib/config.go

```diff
@@ -8,24 +8,27 @@ package lib
 import (
 	"errors"
 	"flag"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
+
+	"gopkg.in/yaml.v2"
 )
 
 // Config configures a deployment.
 type Config struct {
 	conf fileConfig
 
-	AccessKey string
-	SecretKey string
+	AccessKey string `yaml:"key"`
```

Comment:
I'm a little confused by this PR now. To me this looks like a breaking change with limited value (key vs accessKey). I think I will close this PR and we can revisit this some time in the future.